src/surfing_penguin/templates/select_qstnr.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
```

Comment:
What are these blocks for?

Reply:
For consistency, base.html contains the shared content of all html templates, and it has a block region for other templates to store their individual content. Each template based on it just fills in the block content.